dev_scripts/qa.py

```diff
@@ -802,6 +813,31 @@ def _consume_stdin(self):
         while msvcrt.kbhit():
             msvcrt.getch()
 
+    @QABase.task(
+        f"Install the latest version of Python {PYTHON_VERSION_STR}", ref=REF_BUILD
+    )
+    def install_python(self):
+        cur_version = list(sys.version_info[:3])
+
+        logger.info("Getting latest Python release")
+        with urllib.request.urlopen(EOL_PYTHON_URL) as f:
+            resp = f.read()
+            releases = json.loads(resp)
+            for release in releases:
+                if release["cycle"] == PYTHON_VERSION_STR:
+                    latest_version = [int(num) for num in release["latest"].split(".")]
+                    if latest_version > cur_version:
+                        self.prompt(
+                            f"You need to install the latest Python version ({release['latest']})"
```

Comment:
Maybe put `release['latest']` in an intermediate variable?

Reply:
Fixed in b63c1f4f5e1eaf23baebeec7243b977dd9dc1667.